codegen-core/common-test-models/naming-obstacle-course-prelude-ops.smithy

```diff
@@ -0,0 +1,92 @@
+$version: "1.0"
+namespace crate
+
+use smithy.test#httpRequestTests
+use smithy.test#httpResponseTests
+use aws.protocols#awsJson1_1
+use aws.api#service
+use smithy.framework#ValidationException
+
+/// Confounds model generation machinery by using operations named after every item in the Rust prelude
+@awsJson1_1
+@service(sdkId: "Config")
+service Config {
+    version: "2006-03-01",
+    operations: [
+       // Rust Prelude
+       Copy,
+       Send,
+       Sized,
+       Sync,
+       Unpin,
+       Drop,
+       Fn,
+       FnMut,
+       FnOnce,
+       Box,
+       ToOwned,
+       Clone,
+       PartialEq,
+       PartialOrd,
+       Eq,
+       Ord,
+       AsRef,
+       AsMut,
+       Into,
+       From,
+       Default,
+       Iterator,
+       Extend,
+       IntoIterator,
+       DoubleEndedIterator,
+       ExactSizeIterator,
+       Option,
+       Some,
+       None,
+       Result,
+       Ok,
+       Err,
+       String,
+       ToString,
+       Vec,
```

Comment:
`TryFrom`, `TryInto`, and `FromIterator` are missing? https://doc.rust-lang.org/std/prelude/index.html

Would be nice to put the prelude link in these files.

We generate Rust 2021 edition crates.

Reply:
Great catch! Fixed in 49d10aecb5d768cece726b2aaab87a1136852040